pkg/virt-handler/setsched_amd64.go

```diff
@@ -0,0 +1,29 @@
+//go:build linux && amd64
```

Comment:
So I guess this file is duplicated to support multi-arch?

Isn't there a way of doing so without duplicating this file? Maybe something like providing the config to bazel instead of keeping it inside the file?

Reply:
Seems like it is anyway identical at the moment.

How about one file with this build constraint:

```
//go:build (linux && amd64) || ((linux && arm64))
```

Reply:
Thanks for the suggestion @rmohr ! I will push the changes shortly.